doc/DOWNLOADS.md

```diff
@@ -128,3 +118,47 @@ A stable macOS installer for Nicotine+ is available on macOS Catalina 10.15 and
 *NOTE: You have to follow [these instructions](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac) the first time you open Nicotine+ on macOS.*
 
 - [Download Nicotine+ macOS Installer](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/macos-installer.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/macos-installer.zip.sha256)]
+
+## Cross-Platform
+
+### pip
+
+Running Nicotine+ using pip requires a [MinGW environment](https://en.wikipedia.org/wiki/MinGW) on Windows and [Homebrew](https://brew.sh/) on macOS. Note that problems encountered using this method on OSes other than GNU/Linux variants are unlikely to receive support.
+
+To install the latest stable build of Nicotine+ using [pip](https://pip.pypa.io/), ensure the [runtime dependencies](DEPENDENCIES.md) are installed, and run the following:
```

Comment:
Revert sentence. We use "latest stable version" terminology for end-users, and "latest unstable build" for testers.